openmdao/recorders/tests/test_sqlite_recorder.py

```diff
@@ -38,6 +38,10 @@
 if OPTIMIZER:
     from openmdao.drivers.pyoptsparse_driver import pyOptSparseDriver
 
+# do not show matplotlib plots
+import matplotlib
+matplotlib.use('Agg')
```

Comment:
This could be dangerous at the top level of a file, but  I don't think it hurts in the case of tests because no one will be importing this file.